webapp/sources/api-doc/code_samples/curl/campaigns/allCampaigns.sh

```diff
@@ -0,0 +1 @@
+yo
```

Comment:
I think there was a bad copy/paste or something

Reply:
It's part of 'not being finished'